test/src/NoteDebug_test.cpp

```diff
@@ -21,7 +21,7 @@ namespace
 {
 
 typedef struct {
-    char debugBuf[32];
+    char debugBuf[256];
```

Comment:
Is it possible to `#include "note.h"` here, or will it go haywire?

```suggestion
    char debugBuf[NOTE_C_DEBUG_W_LEVEL_MAX_BYTES];
```